apps/blog/src/posts/pwa-inking/pwa-inking.md

```diff
@@ -17,7 +17,7 @@ tags:
   - Windows
 ---
 
-<iframe tabindex="-1" title="GIF of the pwa-inking component working" src="https://giphy.com/embed/gfrRt9TzYCv24qaoCl" width="480" height="308" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/gfrRt9TzYCv24qaoCl">via GIPHY</a></p>
+<iframe title="GIF of the pwa-inking component working" src="\posts\pwa-inking\pwa-blog-2d-ink.gif" width="480" height="308" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/gfrRt9TzYCv24qaoCl">via GIPHY</a></p>
```

Comment:
this should not be an iframe, it should be replaced with an image

Reply:
This is what it looks like right now: 
![image](https://user-images.githubusercontent.com/711864/172740505-378b6694-9584-403e-a4f4-d329f9a4392e.png)

The "Via GIPHY" should also be removed